alkira/resource_alkira_credential_azure_vnet.go

```diff
@@ -24,21 +28,33 @@ func resourceAlkiraCredentialAzureVnet() *schema.Resource {
 				Description: "The Application ID",
 				Type:        schema.TypeString,
 				Required:    true,
+				DefaultFunc: schema.EnvDefaultFunc(
```

Comment:
Why are these parameters on different lines? It shouldn't be too long of a function on the same line.

Reply:
follow the same convention as other providers... gofmt doesn't pick it and then I am okey with it.